HerPortal/Program.cs

```diff
@@ -26,8 +28,17 @@ public static void Main(string[] args)
             using var scope = app.Services.CreateScope();
             var dbContext = scope.ServiceProvider.GetRequiredService<HerDbContext>();
             dbContext.Database.Migrate();
+            
+            // Run nightly tasks at 07:00 UTC daily
```

Comment:
Do we want it daily including weekends? It feels a bit spammy that a submission on a Friday will likely generate 3 emails (on saturday, sunday and monday morning)...

I wonder if we should make the cron configurable so it can be changed more easily later. Hardcoding is probably ok for now though. (This might also make it easier to test!)

Reply:
I asked Kay, and she said sending even over the weekend is what we want 😊 we're strongly erring on the side of sending too many emails, because it's particularly bad if one gets missed!

Yes, I can see the advantage of making it configurable, I'll change it!